src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int finalResult = (result1 == bet1 && result2 == bet2)
+                ? 2 : (result1 == result2 && bet1 == bet2)
+                ? 1 : (result1 > result2 && bet1 > bet2)
+                ? 1 : (result1 < result2 && bet1 < bet2)
+                ? 1 : 0;
+        return finalResult;
```

Comment:
do we need to create separate variable, or can we return result without it?

Reply:
No, we don't need to create, but with the new variable this code more understandable in my humble opinion.